app/Appointments.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Appointments extends Model
+{
+    protected $fillable = [
+        'client_id', 'date_time','type','price',
```

Comment:
Lets get these on a new line: 
```suggestion
        'client_id', 
        'date_time',
        'type',
        'price',
```